ext/excluded_modules.c

```diff
@@ -6,16 +6,23 @@
 #include <ext/standard/php_versioning.h>
 
 #include <components/log/log.h>
+#include "configuration.h"
 
 bool ddtrace_is_excluded_module(zend_module_entry *module, char *error) {
+    if (strcmp("ionCube Loader", module->name) == 0) {
```

Comment:
`#if PHP_VERSION_ID >= 80000` - on PHP 7 it's working to my knowledge - it's not crashing, but instrumentation of the encrypted code itself will not trigger. But that can be fine, as often e.g. framework code isn't encrypted, just the app code.

Reply:
Updated